src/reducers/index.js

```diff
@@ -13,8 +13,18 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+      return {
+        ...state,
+        count: state.count +1,
+      }
+
     case DECREMENT:
     // Fill in the body of this case
+    return {
+      ...state,
+      count: state.count -1,
```

Comment:
good job  keeping the pure functionality by making sure the rest of state (if any existed at this time) are being spread in to avoid mutation